src/Portal.js

```diff
@@ -93,7 +113,12 @@ class Portal extends React.Component {
   }
 
   render() {
-    return null;
+    if (useCreatePortal && this.props.children) {
```

Comment:
paired with my comment above, this needs to confirm `this._overlayTarget` exists for the SSR case

Reply:
done